src/stories/RadioCheckbox.stories.tsx

```diff
@@ -0,0 +1,235 @@
+import { useState } from 'react';
+import RadioCheckboxGroup from 'components/RadioCheckboxButton';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import Form from 'react-bootstrap/Form';
+
+const milkList: { [key: string]: { name: string } } = {
+  'milk-1': { name: 'Whole Milk' },
+  'milk-2': { name: '2% Milk' },
+  'milk-3': { name: 'Almond Milk' },
+};
+
+const toppingsList: { [key: string]: { name: string; price: number } } = {
+  'topping-1': { name: 'Boba', price: 0.5 },
+  'topping-2': { name: 'Popping Boba', price: 0.5 },
+  'topping-3': { name: 'Aloe Vera', price: 0.5 },
+};
+
+export default {
+  title: 'Bobax/RadioCheckboxGroup',
+  component: RadioCheckboxGroup,
+} as ComponentMeta<typeof RadioCheckboxGroup>;
+
+const Template: ComponentStory<typeof RadioCheckboxGroup> = (args) => (
+  <RadioCheckboxGroup {...args} />
+);
+
+type Args = {
+  type: 'radio' | 'checkbox';
+  id: string;
+  label: string;
+};
+export const SingleRadioButton = (args: Args) => {
+  const [checked, setChecked] = useState(false);
+
+  const handleChange = () => {
+    setChecked(true);
+  };
+
+  return (
+    <Form>
+      <RadioCheckboxGroup check={checked} handleChange={handleChange} {...args} />
+    </Form>
+  );
+};
+
+SingleRadioButton.args = {
+  type: 'radio',
+  id: 'milk-1',
+  label: 'Oat Milk',
+};
+
+SingleRadioButton.argTypes = {
+  type: {
+    control: {
+      type: 'radio',
+    },
+  },
+  label: {
+    control: 'text',
+  },
+};
+
+export const SingleCheckBoxButton = (args: Args) => {
```

Comment:
Consistency with Checkbox from component

Reply:
can you explain this part , did you mean renaming it to SingleCheckboxButton ?